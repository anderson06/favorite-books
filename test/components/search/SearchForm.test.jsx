import React from 'react';
import SearchForm from '../../../src/components/search/SearchForm.jsx';
import renderer from 'react-test-renderer';

describe('<SearchForm />', () => {
  it('renders correctly', () => {
    const tree = renderer.create(<SearchForm />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});

