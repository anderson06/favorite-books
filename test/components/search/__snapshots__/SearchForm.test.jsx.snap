// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<SearchForm /> renders correctly 1`] = `
<form
  className="fb-search"
  id="search-form"
  onSubmit={[Function]}
>
  <input
    autoComplete="off"
    className="fb-search-input"
    id="search"
    name="search"
    onChange={[Function]}
    placeholder="Book Name"
    required="required"
    type="search"
  />
</form>
`;
